src/cheri/zysentry.adoc

```diff
@@ -0,0 +1,62 @@
+[#section_rvy_sentry_jalr_ext,reftext="{rvy_sentry_jalr_ext_name}"]
+== "{rvy_sentry_jalr_ext_name}" Extension for CHERI Sentry Capabilities
+
+#Begin new since last ARC review#
+
+[#sentry_cap,reftext="sentry capability"]
+=== Sentry Capabilities
+
+It is useful to have *immutable* function pointers within a CHERI software system.
+<<sealed_cap,Sealed capabilities>> are a natural foundation;
+sealed capabilities which become unsealed via <<JALR_CHERI>> are dubbed "sentries"
+(a portmanteau of "sealed entries").
+Sentry capabilities can establish a form of control-flow integrity between mutually distrusting code.
+
+=== Added Capability Type (CT) Semantics
+
+With {rvy_sentry_jalr_ext_name}, capability encodings _may_ define:
+
+1. Architectural interpretation of zero or more particular <<sec_cap_type>>
+   values when a capability is passed as the input to <<JALR_CHERI>>.
+
+2. Zero or more <<sec_cap_type>> values for use by <<JALR_CHERI>> to seal the
+   return capability installed in the link register.
+
+3. Conditional behavior of <<JALR_CHERI>> based on the _register selectors_
+   used in an instance of the instruction and the <<sec_cap_type>> value of the
+   input capability.
+
+NOTE: All type(s) defined for sealing return capabilities must be acceptable
+for unsealing as inputs to <<JALR_CHERI>> as well (in at least some conditions).
+
+NOTE: Unless other mechanisms to construct sealed capabilities are available,
+all type(s) defined for sealing return capabilities should be
+<<sec_cap_type_ambient,ambiently available>> for (re)construction by <<CBLD>>.
```

Comment:
I think it would be nice to have all of this in the base chapter since this is essentially all a hook for other extension. Maybe we can get by with just adding a note that the instructions to create sentry types are defined by separate extensions.